### PR TITLE
Add glog package.

### PR DESCRIPTION
**Pull request by nickjmeyer**
## Description

Adds the Google Logging library (aka Glog).

## Addtional information
This has only been tested on x86_64.

Works properly:
- [x] x86_64
- [ ] aarch64

### CODE REVIEW

File: packages/glog.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+class Glog < Package
+  description 'A C++ implementation of the Google logging module.'
+  homepage 'https://github.com/google/glog'
+  version '0.3.5'
+
+  source_url 'https://github.com/google/glog/archive/v0.3.5.zip'
+  source_sha256 '267103f8a1e9578978aa1dc256001e6529ef593e5aea38193d31c2872ee025e8'
+
+  depends_on 'openblas'
```
Comment (uberhacker) [inline packages/glog.rb]: Missing `depends_on 'unzip' => :build`?

File: packages/glog.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    system "./configure",
+           "--prefix=#{CREW_PREFIX}",
+           "--libdir=#{CREW_LIB_PREFIX}"
+    system "make"
+  end
+
+  def self.install
+    system "make",
+           "DESTDIR=#{CREW_DEST_DIR}",
+           "LIBDIR=#{CREW_LIB_PREFIX}",
```
Comment (uberhacker) [inline packages/glog.rb]: LIBDIR shouldn't be necessary here since it was already provided in the build section.

File: packages/glog.rb (hunk 1 above)
Comment (nickjmeyer) [reply packages/glog.rb]: Fixed

File: packages/glog.rb (hunk 2 above)
Comment (nickjmeyer) [reply packages/glog.rb]: Fixed

**Pull request merged**